sourcemash/frontend/static/js/views/item_card_view.js

```diff
@@ -31,19 +29,18 @@ Sourcemash.Views.ItemCardView = Backbone.View.extend({
 		this.model.save({vote: -1, voteSum: this._getNewVoteSum(-1)},
                         {success: this.voted});
 
-        if (this.model.changedAttributes()) {
-            mixpanel.track("Downvoted", { "Item Title": this.model.get('title'),
-                                        "Feed Title": this.model.feed.get('title') })
-        }
+        mixpanel.track("Downvoted", { "Item Title": this.model.get('title'),
+                                    "Feed Title": this.model.feed.get('title') })
 	},
 
     voted: function() {
         toast("Vote recorded!", 3000);
     },
 
-    showSubscribeModal: function(e) {
+    showSubscribeModal: function(options) {
         $('#subscribe-modal #unsubscribed-item-title').html(this.model.get('title'));
         $('#subscribe-modal #unsubscribed-feed-title').html(this.model.feed.get('title'));
+        $('#subscribe-modal #unsubscribed-source-title').html(options['source']);
```

Comment:
Sorry to do this to you, but:

would it make more sense to leave event and use a tag in the html to store the source? Then we could access the source like this: http://stackoverflow.com/questions/7823556/passing-parameters-into-the-backbone-events-object-of-a-backbone-view